test/Pester/PlatyPs.Tests.ps1

```diff
@@ -14,12 +14,12 @@ Describe 'New-MarkdownHelp' {
 
     Context 'errors' {
         It 'throw when cannot find module' {
-            { New-MarkdownHelp -Module __NON_EXISTING_MODULE -OutputFolder TestDrive:\ } | 
```

Comment:
Can you, please, separate whitespace changes? Different commit will do but PR is better